### ISSUE TITLE
[BitBucketServer] Missing lines_of_code

### ISSUE THREAD

**Issue by zdenektopic** [danger/danger-js#545]
Hello,

one thing I am missing is `lines_of_code` that was available in Ruby version of Danger. It helps to warn reviewers that PR is bigger than is standard or expected.

**Comment by orta** [comment]
Makes sense, I think it should be pretty easy to add this on to the git DSL. It wasn't added because I think GitHub has it in the PR metadata.
